examples/vue-todos/src-tauri/src/main.rs

```diff
@@ -3,15 +3,52 @@
   windows_subsystem = "windows"
 )]
 
-use tauri::{Menu, MenuItem, Submenu};
+use tauri::{
+  CustomMenuItem, Manager, Menu, MenuItem, Submenu, SystemTray, SystemTrayEvent, SystemTrayMenu,
+  SystemTrayMenuItem,
+};
 use tauri_plugin_sql::{Migration, MigrationKind, TauriSql};
 
 fn main() {
-  let submenu = Submenu::new("File", Menu::new().add_native_item(MenuItem::Quit));
+  let quit = CustomMenuItem::new("quit".to_string(), "Quit");
+  let hide = CustomMenuItem::new("hide".to_string(), "Hide");
+  let show = CustomMenuItem::new("show".to_string(), "Show");
+  let tray_menu = SystemTrayMenu::new()
+    .add_item(show)
+    .add_item(hide)
+    .add_native_item(SystemTrayMenuItem::Separator)
+    .add_item(quit);
+  let tray = SystemTray::new().with_menu(tray_menu);
+
+  let submenu = Submenu::new(
+    "File",
+    Menu::new()
+      .add_native_item(MenuItem::Hide)
+      .add_native_item(MenuItem::Quit),
+  );
   let menu = Menu::new().add_submenu(submenu);
 
   tauri::Builder::default()
     .menu(menu)
+    .system_tray(tray)
+    .on_system_tray_event(|app, event| match event {
+      SystemTrayEvent::MenuItemClick { id, .. } => match id.as_str() {
+        "show" => {
+          let window = app.get_window("main").unwrap();
+          window.show().unwrap();
+          window.set_focus().unwrap();
+        }
+        "hide" => {
+          let window = app.get_window("main").unwrap();
+          window.hide().unwrap();
+        }
+        "quit" => {
+          std::process::exit(0);
```

Comment:
```suggestion
          app.exit(0);
```
Probably better to use this one as it runs a cleanup function before exiting (and is more consistent with other examples/docs).